src/wpt/harness/harness.ts

```diff
@@ -685,6 +685,7 @@ globalThis.assert_not_own_property = (
   );
 };
 
+globalThis.includedFiles = new Set();
```

Comment:
Can you add a comment to what this set does?

Reply:
I moved some stuff around in the harness. Here's where the comment was added:
https://github.com/cloudflare/workerd/blob/2cab4bb9dd84ab2b3d0ab6142e6669177aca8748/src/wpt/harness/harness.ts#L852